web/server/vue-cli/src/components/Report/ReportFilter/Filters/BaselineRunFilter.vue

```diff
@@ -1,40 +1,159 @@
 <template>
   <select-option
     :id="id"
-    title="Run Filter"
+    title="Run / Tag Filter"
     :bus="bus"
     :fetch-items="fetchItems"
     :selected-items="selectedItems"
     :search="search"
     :loading="loading"
+    :apply="apply"
+    @select="prevSelectedRuns = $event"
     @clear="clear(true)"
-    @input="setSelectedItems"
+    @on-menu-show="onMenuShow"
   >
-    <template v-slot:icon>
-      <v-icon color="grey">
-        mdi-play-circle
-      </v-icon>
+    <template v-slot:append-toolbar-title>
+      <selected-toolbar-title-items
+        v-if="selectedToolbarTitleItems.length"
+        :value="selectedToolbarTitleItems"
+      />
+    </template>
+
+    <template
+      v-slot:menu-content="{ items, prevSelectedItems, cancel, select }"
+    >
+      <v-tabs
+        v-model="tab"
+        fixed-tabs
+        hide-slider
+      >
+        <v-tab>Select Runs</v-tab>
+        <v-tab
+          :disabled="prevSelectedRuns.length === 0"
+          class="tags"
+          @click="specifyTagsTab"
+        >
+          Specify Tags
```

Comment:
I have another idea what if the tag and the run selector are called the same way?
```suggestion
          Select Tags
```

Can we change the background of the currently active tab (select run or tag) to slightly different color? That way it is much easier to find out which tab did I select.